src/Directory.build.props

```diff
@@ -4,6 +4,7 @@
     <NoWarn>$(NoWarn);1591;1701;1702;1705;VSX1000</NoWarn>
     <Platform>AnyCPU</Platform>
     <IsTestProject>$(MSBuildProjectName.Contains('Tests'))</IsTestProject>
+    <IsTestHelperProject>$(MSBuildProjectName.Contains('Testing'))</IsTestHelperProject>
```

Comment:
Testing is actually released as a NuGet package. It contains testing utilities for client's.

Reply:
https://www.nuget.org/packages/ReactiveUI.Testing/

Reply:
doh ok i'll remove that